pyvista/plotting/plotting.py

```diff
@@ -1796,7 +1798,7 @@ def remove_actor(self, actor, reset_camera=False):
     def add_actor(self, uinput, reset_camera=False, name=None, loc=None,
                   culling=False, pickable=True):
         """Add an actor to render window.
-        
```

Comment:
@GuillaumeFavelier, can we trailing whitespace to a pycodestyle check?

Reply:
We might need more than just `pydocstyle` for that: http://pycodestyle.pycqa.org/en/latest/intro.html#id3

Reply:
Hm. Then maybe we should just ask all contributors to turn on a setting in their text editor to strip trailing whitespace? For atom, [this works](https://atom.io/packages/whitespace):

![Screen Shot 2019-11-26 at 10 34 58 AM](https://user-images.githubusercontent.com/22067021/69657913-6eb83380-1038-11ea-9bf8-e42dd9733277.png)